style.css

```diff
@@ -74,6 +73,54 @@
   position: absolute;
 }
 
+.footer {
+  background-color: #F5C85F;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  padding: 30px;
+  position: relative;
+  flex-wrap: wrap;
+  box-shadow: 0 -1px 20px 0 rgba(0, 0, 0, 0.3);
+}
+
+.footer::before {
+  height: 10px;
+  content: "";
+  width: 100%;
+  top: -10px;
+  left: 0;
+  position: absolute;
+}
+
+.subscription-title {
+  margin-bottom: 20px;
+  font-size: 16px;
+}
+
+.white-font {
+  color: #FFF;
+}
+
+.subscription-input {
+  background: #FFF;
+  width: 300px;
```

Comment:
always avoid adding a fixed width in pixels if it's greater than 250px this is because it might break in responsive and you'll have to override the width in the media query. a simple way to fix this is to set the width to 100% and set a max width of 300px so the element will have 300px if there are 300px available for it.